.github/workflows/jekyll.yml

```diff
@@ -54,6 +51,24 @@ jobs:
       - name: Upload artifact
         # Automatically uploads an artifact from the './_site' directory by default
         uses: actions/upload-pages-artifact@v1
+      - name: Install Python
```

Comment:
I think it would be better to make this its own separate file since the functionality is separate from making deployments. We could make that other action only run for example when _posts/ is modified so a lot of unnecessary work is avoided. 

If we need to locally build jekyll there to see atom.xml is diffed that would still be fine to do there

Reply:
smh bad devops grindset aadi

tweets are dependent on a successful deploy, and building twice shouldn't happen (much more unnecessary work) -- I think the correct approach is to create a new job in the same file, and make it dependent on the deploy job. It can use `actions/download-artifact` to get the build from the `build` job. That being said, I think this is fine to start, and we can refactor later after we verify that this works correctly

Reply:
You’re right about the job depending on the deployment – I think that approach works. I do think tho it would be good to do it correctly in this one though if possible, as a last step. Anyway flight taking off hit send